### PR TITLE
Add missing Symfony versions to tests

### PR DESCRIPTION
**Pull request by JHGitty**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,6 @@
 language: php
 
 php:
-    - 5.5
```
Comment (K-Phoen) [inline .travis.yml]: Why removing PHP 5.5 ?

File: .travis.yml [hunk 2]
```diff
@@ -16,9 +15,15 @@ env:
 
 matrix:
     include:
-        - php: 5.5
```
Comment (K-Phoen) [inline .travis.yml]: Same here, we do want to test on both the lowest supported Symfony and PHP versions.

File: .travis.yml (hunk 1 above)
Comment (JHGitty) [inline .travis.yml]: We could check against PHP 5.5 **and** PHP 5.6. But why should we check only against PHP 5.5 on Symfony 2.3?

File: .travis.yml (hunk 2 above)
Comment (JHGitty) [inline .travis.yml]: @K-Phoen Thanks! Discussion see other comment.

**Comment by JHGitty** [comment]
@K-Phoen
Should we remove this line? As I understand the matrix includes already contain the Symfony Version. 
Or is it important to have this line still there?

```yml
env:
    - SYMFONY_VERSION="~2.8.0"
```

File: .travis.yml (hunk 1 above)
Comment (K-Phoen) [inline .travis.yml]: See [this PR](https://github.com/dustin10/VichUploaderBundle/pull/470) :)

Comment (JHGitty) [inline .travis.yml]: I understand. Thanks. I'll fix it.

Comment (K-Phoen) [inline .travis.yml]: Awesome, thanks :+1:

**Comment by tgabi333** [comment]
Symfony 2.4, 2.5 and 2.6 are not supported anymore, so there is no need for testing. 

http://symfony.com/doc/current/contributing/community/releases.html

**Comment by K-Phoen** [comment]
@tgabi333 I know, but this bundle claims to support Symfony ~2.3 so I it make sense to test 2.4, 2.5 and 2.6 ;)

**Comment by K-Phoen** [comment]
Closing this since it's solved in master (see #508)

**Pull request closed without merging**